projects/vdk-plugins/vdk-structlog/tests/test_plugin.py

```diff
@@ -1,33 +1,81 @@
 # Copyright 2021-2023 VMware, Inc.
 # SPDX-License-Identifier: Apache-2.0
 import os
-import pathlib
 from unittest import mock
 
 from click.testing import Result
 from vdk.plugin.structlog import structlog_plugin
-from vdk.plugin.test_utils.util_funcs import cli_assert_equal
 from vdk.plugin.test_utils.util_funcs import CliEntryBasedTestRunner
 from vdk.plugin.test_utils.util_funcs import jobs_path_from_caller_directory
 
-"""
-This is a sample test file showing a recommended way to test new plugins.
-A good way to test a new plugin is how it would be used in the command that it extends.
-"""
 
+def test_structlog():
```

Comment:
Can you also add a similar test using the JsonFormatter? Filtering is a little bit different there, so it should also be tested end-to-end.